docs/detections/rules-ui-create.asciidoc

```diff
@@ -136,6 +150,55 @@ per rule execution:
 ** _Field_: Leave blank
 ** _Threshold_: `10`
 
+* To create a new EQL rule, select _Event Correlation_ and then:
+.. Define which {es} indices the rule searches for alerts.
+.. Add an {ref}/eql-syntax.html[EQL statement] used to detect alerts.
++
+For example the following rule detects when `msxsl.exe` makes an outbound
+network connection:
+
+** _Index patterns_: `winlogbeat-*`
++
+Winlogbeat ship Windows events to {es-sec}.
+** _EQL query_: `sequence by process.entity_id [process where event.type in ("start", "process_started") and process.name == "msxsl.exe"] [network where event.type == "connection" and process.name == "msxsl.exe" and network.direction == "outgoing"]`
```

Comment:
Is there a way to make this a multiline code block? It's a bit hard to read as it is. Something like

```
sequence by process.entity_id
  [process 
    where event.type in ("start", "process_started") 
    and process.name == "msxsl.exe"]
  [network 
    where event.type == "connection" 
    and process.name == "msxsl.exe"
    and network.direction == "outgoing"]
```

Not sure if that's the "correct" EQL formatting, but you get the idea 🤷‍♂️

Reply:
@rylnd: I think so, let me explore this one.